### ISSUE TITLE
Lesson does not render

### ISSUE THREAD

**Issue by hlapp** [datacarpentry/cloud-genomics#14]
Accessing the rendered version of this lesson, both through the [URL given by Github](http://datacarpentry.github.io/cloud-genomics/) and by the [datacarpentry.org custom domain](http://www.datacarpentry.org/cloud-genomics/), yield 404s.

It's not clear to me what the reason is? I do notice that apparently this is using redcarpet for Markdown rendering, [which is being phased out by Github](https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0).

**Comment by tracykteal** [comment]
It's because there's not an index.md There's a PR in to fix that, but I'd like to get the _site directory out of the PR first. I can merge though and then fix.

**Comment by tracykteal** [comment]
Also, all the lesson use red carpet, so we have to go make that change throughout.

**Comment by mkuzak** [comment]
I'd be in favour of migrating to new styles
https://github.com/swcarpentry/styles

**Comment by JasonJWilliamsNY** [comment]
Yes - I think its actually painful to go forward without fixing that first
